lib/MessageBox.js

```diff
@@ -1,15 +1,26 @@
-import Handlebars from 'handlebars';
 import deepExtend from 'deep-extend';
+import template from 'lodash.template';
 
 class MessageBox {
   constructor({
     initialLanguage,
     messages,
     tracker,
+    interpolate,
+    evaluate,
+    escape,
   } = {}) {
     this.language = initialLanguage || MessageBox.language || 'en';
     this.messageList = messages || {};
     if (tracker) this.trackerDep = new tracker.Dependency();
+
+    // Default lodash templates regexs
+    // https://regex101.com/r/ce27tA/5
+    this.interpolate = interpolate || MessageBox.interpolate || /{{{([^\{\}#][\s\S]+?)}}}/g;
+    // https://regex101.com/r/ndDqxg/4
+    this.evaluate = evaluate || MessageBox.evaluate || /{{#([^\{\}].*?)}}/g;
+    // https://regex101.com/r/8sRC8b/8
+    this.escape = escape || MessageBox.escape || /{{([^\{\}#][\s\S]+?)}}/g;
```

Comment:
Rather than adding the default expression here, can you set initial values for the static props? So something like

```js
this.interpolate = interpolate || MessageBox.interpolate;
this.evaluate = evaluate || MessageBox.evaluate;
this.escape = escape || MessageBox.escape;
```

And after the constructor function:

```js
static interpolate = /{{{([^\{\}#][\s\S]+?)}}}/g;
static evaluate = ...
```

And so on for all three.